app/controllers/templates_controller.rb

```diff
@@ -33,7 +33,7 @@ def update
   end
 
   def show_form
-    @template = Template.find params[:id]
+    @template = Template.find params[:id] || Template.where(:active => true).first
```

Comment:
Not sure we should do a fail state like this. I think it probably makes more sense to just throw an error if you somehow try to access the form for a template that doesn't exist. Interested in hearing your thoughts on why you did it this way.